Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesAB.swift

```diff
@@ -622,21 +614,17 @@ public struct AccessorEffectSpecifiersSyntax: SyntaxProtocol, SyntaxHashable, _L
     }
   }
   
-  /// The `throws` keyword.
-  ///
-  /// ### Tokens
-  /// 
-  /// For syntax trees generated by the parser, this is guaranteed to be `throws`.
-  public var throwsSpecifier: TokenSyntax? {
+  /// The clause specifying thrown errors
+  public var throwsClause: ThrowsClauseSyntax? {
```

Comment:
This needs a compatibility layer.

Reply:
It's handled via the extension of the EffectSpecifiersSyntax protocol.

Reply:
Ah, I missed that `EffectSpecifiersSyntax` was a protocol. 👍🏽